callback-free-itx/script.ts

```
@@ -33,12 +34,13 @@ const prisma = new PrismaClient({ log: ["query"] }).$extends({
       ) {
         const tx = prisma.$transaction((txClient) => {
           setTxClient(txClient as unknown as Prisma.TransactionClient);
-
-          return txPromise.catch((e) => {
-            if (e === ROLLBACK) return;
```

this swallows the exception that actually triggers the rollback.